MADRaS/envs/data/sim_options.yml

```diff
@@ -1,15 +1,26 @@
 torcs_server_port: 60934
+server_config:
+  max_cars: 10
+  track_names:
+    - forza
+    - aalborg
+    - alpine-1
+    - alpine-2
+  distance_to_start: 25
+  torcs_server_config_path: /home/anirban/.torcs/config/raceman/  # This must be the value for TORCS with rendering on
```

Comment:
Specific path, let's just do `<username>` instead so that users know what to put in.

Reply:
Done.